d2l-button-icon.html

```diff
@@ -21,10 +21,11 @@
 				--d2l-button-icon-border-radius: 0.3rem;
 				--d2l-button-icon-min-height: calc(2rem + 2px);
 				--d2l-button-icon-min-width: calc(2rem + 2px);
+				--d2l-button-icon-h-align: calc(((2rem + 2px - 0.9rem) / 2) * -1);
```

Comment:
Sorry for the delay, catching up on reviews... I think this change may break anywhere that was setting a custom `--d2l-button-icon-min-width`, like https://github.com/BrightspaceUI/inputs/blob/03eac69eb4cef115a2d4183b65c2eb886df105a5/d2l-input-search.html#L70.

They can fix it by setting `--d2l-button-icon-h-align` appropriately, but if this is a minor release then I think search input may be broken

Reply:
Doesn't look like they're using h-align.

Reply:
Ah good point.  I did a quick check against the three GitHub repos and I'm not seeing anywhere that sets a `--d2l-button-icon-min-width` and uses `h-align`